docs/apps/components/collapse.md

```diff
@@ -0,0 +1,124 @@
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of an alert in response to user input.
+    2. **fields** refer to the component **ID**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. For using **this.show(fields)** enable the **Hidden** check-box in the **Component Attribute** panel.
+
+#### Steps to place/use the methods for components of the Page Builder"
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action')
+2. Write the method/code as shown in the **Example** below
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/alert1.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```js
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps in the above **Note**
+          * ```js
+            this.addClassName('collapse_itxbeisp', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('collapse_itxbeisp', 'abc')
+        ```
+    
+    3. `this.hide(fields)`
+        ```js
+        var fields= ['collapse_itxbeisp']
+        this.hide(fields)
+        ```
+    
+    4. `this.show(fields)`
+        ```js
+        var fields= ['collapse_itxbeisp']
+        this.show(fields)
+        ```
+    
+    5. `this.disable(['fields'])`
+        ```js
+        this.disable(['collapse_itxbeisp'])
+        ```
+    
+    6. `this.getValue('fieldName')`
+        ```js
+        var collapsename = this.getValue('collapse_itxbeisp');
+        console.log('getValue', collapsename);
+        ```
+    
+    7. `this.getComponent('component_ID')`
+        ```js
+        var collapsename = this.getComponent('collapse_itxbeisp');
+        console.log('getComponent', collapsename);
+        ```
+
+## Config
+
+| **Name**|**Description**|**Icon**|
+|---------|---------------|--------|
+|**ID**| Collapse's unique identifier|<img src= "/apps/components/img/input_id.png">|
+|**Collapse Options**|Allows you to add/delete multiple Collapses|<img src= "/apps/components/img/collapse_collapseoptions.png">|
+|**Accordian Mode**|Enable this mode if you want to display only one Collaspse section at a time (this is helpful when you have to avoid clutter when you have multiple Collapse sections)|<img src= "/apps/components/img/collapse_accordionmode.png">
+|**Custom Class**| An HTML class attribute which allows further customisation **[See Form Attribute > Style Sheets**](https://bani-appsection--connexcs-docs.netlify.app/apps/page-builder/#form-attribute)|<img src= "/apps/components/img/input_customclass.png">|
```

Comment:
The double asterisk surrounding the link (See Form Attribute > Style Sheets) should be removed.

Reply:
removed